src/App.vue

```diff
@@ -50,9 +50,9 @@ if (query) {
   flex-direction: column;
   min-height: 100vh;
   align-items: center;
-  margin-left: 1rem;
-  margin-right: 1rem;
-  padding-bottom: 3rem;
+  margin-left: 3.75rem;
+  margin-right: 3.75rem;
+  padding-bottom: 6.25rem;
```

Comment:
Out of curiosity, how does it look if we use the lux variable `--space-x-large` instead of `3.75rem` and `--space-xx-large` instead of `6.25rem`?  It just would be nice to use more of lux, but only if it looks okay.

Reply:
I'll add new variables in lux so that they much with the figma design.